inc/integrations/class-pico.php

```diff
@@ -91,6 +101,19 @@ public function render_pico_auth_key_input() {
 		<?php
 	}
 
+	/**
+	 * Render an input for the Pico Whitelisted SSO Tiers.
+	 */
+	public function render_pico_tiers_input() {
+		// Check to see if there are existing tiers in the option.
+		$tiers = $this->options[ $this->option_key ]['tiers'] ?? '';
+
+		?>
+			<input type="text" name="irving_integrations[<?php echo esc_attr( 'pico_tiers' ); ?>]" value="<?php echo esc_attr( $tiers ); ?>" />
```

Comment:
We may want to add some help text that explains that this needs to be a comma separated list.